tests/org.eclipse.n4js.ui.tests/src/org/eclipse/n4js/dirtystate/CanLoadFromDescriptionPluginUITest.xtend

```diff
@@ -157,6 +156,13 @@ class CanLoadFromDescriptionPluginUITest extends AbstractCanLoadFromDescriptionT
 		assertMarkers("file B2 should have no errors", fileB2, 0)
 		assertMarkers("file C should have no errors", fileC, 0)
 	}
+	
+	/* Same as the super impl but without the super expensivce waitForAutobuild */	
```

Comment:
expensivce

Reply:
Do you recommend that we should remove the expensive call to waitForAutobuild in the original method?

Reply:
In general: yes, to speed up test execution.